reTurn/client/TurnAsyncSocket.cxx

```diff
@@ -828,7 +829,7 @@ TurnAsyncSocket::handleBindRequest(StunMessage& stunMessage)
    StunMessage::setStunAtrAddressFromTuple(response->mXorMappedAddress, stunMessage.mRemoteTuple);
 
    // Add Software Attribute
-   response->setSoftware(SOFTWARE_STRING);
+   response->setSoftware(getSoftware());
```

Comment:
Further to comment below about not adding software header if set to empty, these set lines could be changed to the following and we can remove getSoftware():

if(!mSoftware.empty())
{
    response->setSoftware(mSoftware.c_str());
}